static/images/clients/finamp.svg

```diff
@@ -1,13 +1,121 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 21.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 viewBox="0 0 237.9 238" style="enable-background:new 0 0 237.9 238;" xml:space="preserve">
-<style type="text/css">
-	.st0{fill:none;}
-	.st1{opacity:0.72;fill:#FFFFFF;}
-</style>
-<rect class="st0" width="237.9" height="237.9"/>
-<polygon class="st1" points="126.8,205.6 126.7,205.6 80.1,159 106.7,132.3 179.9,205.6 "/>
-<polygon class="st1" points="126.7,112.3 126.7,112.3 80,158.9 106.7,185.5 179.9,112.3 "/>
-<polygon class="st1" points="126.7,32.5 126.6,32.4 39.9,119 66.6,145.7 179.8,32.5 "/>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
You can save an "optimized svg" in inkscape. This will remove all metadata (like your windows path `C:\Users\Sivilian\Pictures\Finamp.png`) and make a smaller output.